tests/mocks/index.ts

```diff
@@ -0,0 +1,10 @@
+import { anything, instance, mock, when } from 'ts-mockito';
+import StorageService from '../../services/StorageService';
+
+export default class Mocks {
+  public static storage(fileLocation: string) {
```

Comment:
Inline this and use `verify` to check that we called the mock correctly.

Reply:
I'm not too sure what you mean by this. What should we be inlining, and where should we be verifying the call?